client/src/components/PinnedFAQ/PinnedFAQ.js

```diff
@@ -0,0 +1,41 @@
+import _ from "lodash";
+import React from "react";
+
+import { KeyConceptList } from "src/components/KeyConceptList/KeyConceptList";
+
+import { Gov } from "src/models/organizational_entities";
+
+import { secondaryColor } from "src/style_constants/colors.interop.scss";
+
+import { SomeThingsToKeepInMind } from "./faq_utils";
```

Comment:
I think `SomeThingsToKeepInMind` doesn't need to be a standalone component now, I don't think there's anything using it directly (only via the `PinnedFAQ`). The I think merging it directly with `PinnedFAQ` would be good.